engine/routines/flagpredef.asm

```diff
@@ -1,5 +1,5 @@
-Predef_FlagAction: ; 4d7c1
-; Perform action b on flag c in flag array hl.
+Predef_SmallFarFlagAction: ; 4d7c1
```

Comment:
This is not always far, only when d = 1.

Reply:
I know... it's kinda like the FarCopyBytes kinda functions. You can use them to copy a byte from the current bank. I know, there's always a bankswitch in those regardless, but you are still forced to set d to the bank argument here as well (even if it's just setting it to 0 to avoid bankswitch).
I mean, I don't really care. I can rename it again if wanted.

Reply:
Eh, it's fine. Come to think of it, `FlagAction` does *not* support bank switching, so "SmallFar" accurately describes this one's differences.

I would, however, either rename the file to predef_smallfarflagaction.asm (ugly) or find a better place for it than engine/routines/*lowercaselabelname*.asm (which was never meant to be permanent anyway).

Reply:
Yes, the file should be renamed (along with many others), but I don't think it's a good idea to do more in this PR until the current open ones are merged. I know it's just a single file that won't cause any conflict, but it would probably become renamed/moved again when that stuff is dealt with in another PR anyway.